src/foam/nanos/notification/email/SMTPEmailService.js

```diff
@@ -64,6 +67,18 @@ foam.CLASS({
     }
   ],
 
+  axioms: [
+    {
+      name: 'javaExtras',
+      buildJavaClass: function (cls) {
+        cls.extras.push(foam.java.Code.create({
+          data:
+  `FixedThreadPool pool = new FixedThreadPool();`
```

Comment:
There's already a ThreadPool in the context which you should use instead of creating your own.

Reply:
Thanks, fixed